lily.properties

```diff
@@ -29,31 +29,39 @@ logs.desc = Game logs can be found in the `logs` sub-directory in .minecraft on
 rule.children = 1 2 3 4 5 6 7 8 9
 
 rule.child.1.help = Reminds the user of Rule 1: Be decent to one another.
-rule.child.2.help = Reminds the user of Rule 2: Keep chat clean.
-rule.child.3.help = Reminds the user of Rule 3: Keep discussion on-topic.
-rule.child.4.help = Reminds the user of Rule 4: Understand that support is not guaranteed.
-rule.child.5.help = Reminds the user of Rule 5: Do not ask for support on compiling Iris.
-rule.child.6.help = Reminds the user of Rule 6: No links to executable files.
-rule.child.7.help = Reminds the user of Rule 7: Refrain from sending unsolicited pings and DMs.
-rule.child.8.help = Reminds the user of Rule 8: Adhere to the Discord Terms of Service.
-rule.child.9.help = Reminds the user of Rule 9: Don't recommend unofficial forks of Iris to people in support channels.
 rule.child.1.title = Rule 1
-rule.child.2.title = Rule 2
-rule.child.3.title = Rule 3
-rule.child.4.title = Rule 4
-rule.child.5.title = Rule 5
-rule.child.6.title = Rule 6
-rule.child.7.title = Rule 7
-rule.child.8.title = Rule 8
-rule.child.9.title = Rule 9
 rule.child.1.desc = Be decent to one another. We're all human. Any and all forms of bigotry, harassment, doxxing, exclusionary, or otherwise abusive behavior will not be tolerated. Excessive rudeness, impatience, and hostility are not welcome. Do not rage out or make personal attacks against other people. Do not encourage users to brigade/raid other communities.
+
+rule.child.2.help = Reminds the user of Rule 2: Keep chat clean.
```

Comment:
The formatting here got absolutely destroyed, please restore it to how it was previously.

Reply:
I think it's just spaces between each rule which makes a bit of sense but not really something in need of changing